src/components/home/modals/Send.vue

```diff
@@ -490,6 +499,7 @@ export default {
     },
     created() {
         this.getSuperNodes()
+        this.aliasAddressList = JSON.parse(window.localStorage.getItem(this.defaultAddress)).alias
         this.coldRecipientAddressList = new LRUCache(10)
         this.hotRecipientAddressList = new LRUCache(10)
```

Comment:
If `this.aliasAddressList` is not empty, no longer shows address history. Use alias address instead. (`this.coldRecipientAddressList` and `this.hotRecipientAddressList` should  `= this.aliasAddressList`).